app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,36 @@
+<% menu_items.each do |group_name, group| %>
+  <li>
+    <% if group[:children].length == 1 %>
+      <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_controller(request) == group[:children][0][:controller] %>"
+         href="/admin/<%= group[:children][0][:controller] %>"
+         aria-page="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
+      >
+        <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon") %>
+        <%= display_name(group_name) %>
+      </a>
+    <% else %>
+      <button class="crayons-link crayons-link--block cursor-pointer <%= "crayons-link--current" if deduced_scope(request) == group_name.to_s %>"
+        data-toggle="collapse"
+        data-target="#<%= group_name %>"
+        aria-expanded="<%= (deduced_scope(request) == group_name.to_s).to_s %>"
+        aria-controls="<%= group_name %>"
+      >
+        <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
```

Comment:
Minor nitpick 😄 

```suggestion
        <%= inline_svg_tag(group[:svg], aria: true, class: "dropdown-icon crayons-icon", title: group_name) %>
```

Reply:
Thanks for picking this one up @jgaskins! 😊 I removed the .svg into the menu file and forgot to remove the string interpolation ✅